rust/core-lib/src/config.rs

```diff
@@ -16,6 +16,7 @@ use std::collections::HashMap;
 use std::error::Error;
 use std::fmt;
 use std::fs;
+use std::io::ErrorKind;
```

Comment:
you can combine this import into the statement before, so we only import from `std::io` once. :)